Bearsampp/Bearsampp
# Update nssm core to 2.24

This updates nssm to 2.24 x64
To test:
Build and make sure win services like change php version still work.
make sure start/stopping service(s) work.
